packages/swap/src/routes/quote.ts

```diff
@@ -24,6 +41,24 @@ import {
 } from '../utils/rpc';
 import ServiceError from '../utils/ServiceError';
 import { getBrokerQuote } from '../utils/statechain';
+import { estimateSwapDuration } from '../utils/swap';
+
+const getPoolQuoteResult = resultify(getPoolQuote);
```

Comment:
there is `getPoolQuote` and `getBrokerQuote` now which is a bit confusing. maybe we should rename the `getBrokerQuote` to `getSwapRate` to not reuse `quote` and match the rpc naming?